packages/samples/vuejs/src/components/SearchPage.vue

```diff
@@ -0,0 +1,52 @@
+<script setup lang="ts">
+import {onMounted} from 'vue';
+
+interface InitializationOptions {
+  organizationId: string;
+  accessToken: string;
+}
+
+interface HTMLAtomicSearchInterfaceElement extends HTMLElement {
```

Comment:
There should be a way to import that type from Atomic package 🤔 

Something like: 
```
import {Components} from '@coveo/atomic';

// [...]

const searchInterface = document.querySelector('atomic-search-interface') as Components.AtomicSearchInterface;
```

Reply:
I'm currently using the CDN version, any ideas how to use it this way?